sup3r/preprocessing/wind_conditional_moment_batch_handling.py

```diff
@@ -0,0 +1,354 @@
+# -*- coding: utf-8 -*-
+"""
+Sup3r wind conditional moment batch_handling module.
+"""
+import logging
+import tensorflow as tf
+import numpy as np
+
+from sup3r.utilities.utilities import (spatial_simple_enhancing,
+                                       temporal_simple_enhancing)
+from sup3r.preprocessing.batch_handling import Batch
+from sup3r.preprocessing.conditional_moment_batch_handling import (
+    BatchMom1,
+    ValidationDataMom1,
+    BatchHandlerMom1)
+
+np.random.seed(42)
+
+logger = logging.getLogger(__name__)
+
+
+class WindBatchMom1(BatchMom1):
+    """Batch of low_res, high_res and output wind data"""
+
+
+class WindBatchMom1SF(WindBatchMom1):
+    """Batch of low_res, high_res and output wind data
```

Comment:
my OCD forces me to ask if we can only add a line break at the point the text would go passed the 79th column.